packages/docs/content/blog/release-6.0.mdx

```diff
@@ -54,15 +54,15 @@ In our [3.6.0 release](https://github.com/CMSgov/design-system/releases/tag/%40c
   <img src={swatches} alt="Color ramps from the 'System Color Tokens' page, showing a range of light to dark shades of a color" className="ds-u-display--inline-block" width="400" />
 </div>
 
-In this release we've added a page to the design system to show all color tokens in an easy-to-read and scannable way. We're exposing this base set of tokens to better highlight the design decisions made within a particular theme. Think of them as [a complete box of crayons](https://design.cms.gov/foundation/system-color-tokens/?theme=core). Our themes make use of certain colors from the crayon box but not all of them.
+In this release we've added a page to the design system to show all color tokens in an easy-to-read and scannable way. We're exposing this base set of tokens to better highlight the design decisions made within a particular theme. Think of them as [a complete box of crayons](/foundation/system-color-tokens/?theme=core). Our themes make use of certain colors from the crayon box but not all of them.
 
 ### Streamlining the ZIP Code design pattern
 
 We dove deep into a dozen CMS, HealthCare, and Medicare products, and what we discovered was truly eye-opening. We found a wide range of design patterns used for ZIP code input fields, but we also found some commonalities that can lead to a more efficient and user-friendly experience.
 
 Some products used a 5-digit ZIP code pattern, others used a 9-digit ZIP code pattern for stricter data validation. We also discovered services using an autocomplete feature for users to confirm their geo-location.
 
-ZIP codes are an important part of how we search for services and determine eligibility at CMS. Overall, we're thrilled to bring these design insights to the table and excited to see the impact they'll have on our user's experiences. [ZIP code pattern guidance](https://design.cms.gov/patterns/zip-codes/?theme=core).
+ZIP codes are an important part of how we search for services and determine eligibility at CMS. Overall, we're thrilled to bring these design insights to the table and excited to see the impact they'll have on our user's experiences. [ZIP code pattern guidance](/patterns/zip-codes/?theme=core).
```

Comment:
I believe our guidance is links should have action language to tell ppl what to expect, but "[ZIP code pattern guidance]" doesn't adhere to that

Reply:
I'll change it and fix the URL. How about
> See ZIP code pattern guidance?